### DISCUSSION TITLE
Including "type" on the object cards (or in the search function)

### DISCUSSION THREAD

**Discussion by KJBock** [CollectionBuilder/collectionbuilder.github.io#110] [Q&A] [answered]
Hi there, 
I wanted to ask where I would need to make changes if I wanted the "type" of an object displayed on the object card itself. 
I looked in into the browse.js and item.js file but I am unsure where I would need to add that "type" should also be displayed. 
Any pointers would be much appreciated. 
I am trying to include this into the object card, for clarity, and also for the type to be included in the search function - which I think is searching for the keywords on the card only?
Best wishes, Katharina

**Comment by KJBock** [comment, marked as answer]
I have found the solution myself - just in case it it helpful for others:
If you go to config-browse.csv you can simply add it in a field, it will then show up in the card and also be searchable.

**Reply by dcnb** [reply]
Hi @KJBock,  

Nice job! and thanks for sharing the answer! Here's a link to the documentation on configuring the browse page: 

[https://collectionbuilder.github.io/cb-docs/docs/customization/config-browse/](https://collectionbuilder.github.io/cb-docs/docs/customization/config-browse/)
